deploy/kiali/kiali_cr.yaml

```diff
@@ -520,9 +520,10 @@ spec:
 # config_map_name: The name of the istio control plane config map. It defaults to `istio`.
 # envoy_admin_local_port: The port which kiali will open to fetch envoy config data information.
 # istio_identity_domain: The annotation used by Istio to identify domains.
+# istio_injector_name: The name of the istio-sidecar-injector config map. This defaults to `istio-sidecar-injector`
 # istio_injection_annotation: The annotation used by Istio to automatically inject a specific workload
 # istio_sidecar_annotation: The pod annotation used by Istio to identify the sidecar.
-# istiod_deployment_name: The name of the istiod deployment.
+# istiod_deployment_name: The name of the istiod deployment. Default: `istiod`
```

Comment:
Revert this line change for the same reason described in my earlier comment ("We don't add verbosity to these help strings by specifying defaults because all the values you see here ARE the defaults") - and as you see here, this file does document the correct default when it declares: ` istiod_deployment_name: "istiod"`
```suggestion
# istiod_deployment_name: The name of the istiod deployment.
```